Dockerfile

```diff
@@ -1,45 +1,47 @@
-FROM perl:5.38
-
-RUN apt-get update && apt-get install -y \
-  bsd-mailx \
-  msmtp-mta \
-  pigz \
-  ruby-dev
-
-RUN cpanm -n  \
-  Data::Dumper \
-  Date::Manip \
-  DBD::MariaDB \
-  DB_File \
-  DBI \
-  Devel::Cover \
-  Devel::Cover::Report::Coveralls \
-  Exporter \
-  File::Slurp \
-  https://github.com/hathitrust/progress_tracker.git@v0.9.0 \
-  JSON::XS \
-  LWP::Simple \
-  Mail::Mailer \
-  MARC \
-  MARC::Batch \
-  MARC::File::XML \
-  MARC::Record \
-  MARC::Record::MiJ \
-  Sys::Hostname \
-  Test::More \
-  Test::Output \
-  URI::Escape \
-  XML::LibXML \
-  XML::LibXSLT \
-  YAML \
-  YAML::XS
+FROM debian:bookworm
 
 ENV ROOTDIR /usr/src/app
 ENV HOME /usr/src/app
 
 COPY . $ROOTDIR
 WORKDIR $ROOTDIR
 
+# Install Debian packages
+RUN apt-get update && apt-get install -y \
+    bsd-mailx \
+    build-essential \
+    cpanminus \
+    git \
+    libcrypt-ssleay-perl \
+    libdate-manip-perl \
+    libdb-dev \
+    libdbd-mariadb-perl \
+    libdevel-cover-perl \
+    libjson-xs-perl \
+    libmarc-file-mij-perl \
+    libmarc-perl \
+    libmarc-record-perl \
+    libmarc-xml-perl \
+    libmariadb-dev \
+    libnet-ssleay-perl \
+    libssl-dev \
+    libtest-output-perl \
+    libwww-perl \
+    libxml-libxml-perl \
+    libyaml-libyaml-perl \
+    msmtp-mta \
+    perl \
+    pigz \
+    ruby-dev
+
+# Install perl modules that we cannot get with apt-get
+RUN cpanm --notest \
+    DB_File \
+    Devel::Cover::Report::Coveralls \
+    Test::More \
```

Comment:
`Test::More` is part of the `perl-modules-5.36` package which is installed by default when you `apt install perl`

Reply:
yep, trimmed this and it still works